src/signals/settings/users/Detail/__tests__/Detail.test.js

```diff
@@ -520,7 +524,7 @@ describe('signals/settings/users/containers/Detail', () => {
 
     expect(navigateMock).not.toHaveBeenCalled()
 
-    act(() => {
+    await act(async () => {
```

Comment:
not needed

Reply:
removed the unnecessary awaits ...async in this file, there was one that was still necessary